packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -3089,6 +3090,18 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
     });
   }
 
+  @override
+  void performSelector(String selectorName) {
+    final Intent? intent = intentForMacOSSelector(selectorName);
+
+    if (intent != null) {
```

Comment:
Should we assert that the intent isn't null?  I'm thinking that somewhere at least a warning should be logged if no intent was found for the selector.  Or do we expect to be sent weird selectors from MacOS?

Reply:
Uses can define completely custom selectors in DefaultKeyBinding.Dict. Those will be passed along from TextInputContext. I'm not entirely sure what the point of it is, but it definitely shouldn't cause weird assertions.

Reply:
Ah got it.  Agreed, we probably shouldn't warn/error in that case.